activerecord/test/cases/base_test.rb

```diff
@@ -1122,15 +1123,14 @@ def test_no_limit_offset
     end
   end
 
-  def test_find_last
-    last  = Developer.last
-    assert_equal last, Developer.all.merge!(:order => 'id desc').first
-  end
```

Comment:
This test is the inverse of the `test_last` test, for me it's a duplication because if one test fails the other will fail too.